framework/test/gtest/ServiceHooksTest.cpp

```diff
@@ -190,14 +204,16 @@ namespace
             ASSERT_EQ(context, bundleCtx);
 
             references.clear();
-            ordering.push_back(id);
+            ordering_->push_back(id);
         }
 
-        static std::vector<int> ordering;
+        std::vector<int>
```

Comment:
return a `std::shared_ptr<std::vector<int>>>` instead. this breaks the purpose of having a `std::shared_ptr<std::vector<int>>>`